app/models/user.rb

```diff
@@ -10,6 +10,12 @@ class User < ApplicationRecord
   validates :name, presence: true
   validates :posts_count, numericality: { only_integer: true, greater_than_or_equal_to: 0 }
 
+  ROLES = %w[admin user].freeze
+
+  def admin?
+    role == 'admin'
```

Comment:
- [ ] [OPTIONAL] Great job setting the `user.admin` role here :clap: but I think introducing a condition to this so that it doesn't default to true all the time in order to prevent buggy performance would be okay :thinking: 

```
   def admin?
     role == 'admin' ? true : false
   end